resources/profiles/Creality/machine/Creality CR-10 SE 0.2 nozzle.json

```diff
@@ -15,11 +15,11 @@
     ],
     "printable_area": [
       "0x0",
-      "250x0",
-      "250x250",
-      "0x250"
+      "220x0",
+      "220x220",
+      "0x220"
```

Comment:
I measured the bed size as 235x235. So 230x230 should be a good value. If the user accounts for the purge line.

Reply:
In Creality's slicer it says the bed size is 220x220. It's also what is reported in the docs.

Printer.cfg says y goes from -5.5 to 225 and x is -10.5 to 243.

Reply:
Ok Creality seems to play it safe also. I see they reduced X max by 2 mm in printer.cfg.
I will resolve this comment.